### ISSUE TITLE
Sessions: в управлении сессиями показаны только сохранённые вручную сессии

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#3423]
Ставим плагин Sessions. По умолчанию в нём включено сохранение сессии при выходе. Открываем несколько вкладок. Перезапускаем Миранду. Открываем несколько других вкладок. Перезапускаем Миранду.

Теперь у нас есть 2 автосохранённые сессии. Если выбрать Главное меню -> Менеджер сессий -> Восстановить сессию, то мы видим выпадающий список этих автосохранённых сессий. И там же есть кнопка "Уд.", чтобы удалить сессию.

<img width="158" alt="mstsc_LAOw8KHd0n" src="https://user-images.githubusercontent.com/5046855/224562403-b06b30ba-8d1a-49e2-9dfc-d264c91deefd.png">

Но в настройках плагина этих сессий нет

<img width="388" alt="224562193-b6b0263b-392d-443e-adf1-0260294916d4" src="https://user-images.githubusercontent.com/5046855/224562457-2bd5f60d-e2fc-4f89-b642-510fe30016bd.png">

В этот список попадают только сессии, сохранённые вручную, но не автоматически.

Предлагаю выводить там и автосохранённые.

**Comment by georgehazan** [comment]
Нет, в опциях им делать решительно нечего, там настраиваются только кастомные наборы окошек, а не всякая текущая хрень
